core/domain/exp_services.py

```diff
@@ -239,19 +239,21 @@ def get_exploration_by_id(exploration_id, strict=True, version=None):
             return None
 
 
-def get_exploration_summary_by_id(exploration_id):
+def get_exploration_summary_by_id(exploration_id, strict=True):
     """Returns a domain object representing an exploration summary.
 
     Args:
         exploration_id: str. The id of the ExplorationSummary to be returned.
+        strict: bool. If True, an EntityNotFoundError is raised when any
```

Comment:
Should we write `Raises:` in the docstring?

Reply:
I mentioned the error case in "strict" definition. The other functions like get_exploration_by_id uses similar type of docstring